package.json

```diff
@@ -13,7 +13,7 @@
     "@mui/material": "^5.8.7",
     "@mui/styled-engine-sc": "^5.8.0",
     "@mui/styles": "^5.11.2",
-    "@mui/x-date-pickers": "^5.0.0-alpha.7",
+    "@mui/x-date-pickers": "^5.0.17",
```

Comment:
we should be able to remove this and it's usage..  cc @constanzauanini thoughts? looks like some really old code/usage

Reply:
Probably yes, we could try removing it from where it is used